src/OrchardCore.Modules/OrchardCore.Search.Lucene/Services/LuceneIndexManager.cs

```diff
@@ -192,7 +192,7 @@ public void Read(string indexName, Action<IndexReader> reader)
         /// </summary>
         public IReadOnlyDictionary<string, DateTime> GetTimestamps()
```

Comment:
Delete this method, it is not used.

Reply:
@sebastienros done. Are you good with merging this PR?